test/bootstrapUtil.js

```diff
@@ -29,24 +43,35 @@ const wrap = (lam) => (req, res) => {
   return lam(req, res)
 }
 
-export const cleanupUtil = () => {
+const cleanupUtil = () => {
   Object.keys(require.cache)
     .filter((m) => m.includes('src/util') || m.includes('src/lib'))
     .map((m) => delete require.cache[m]) // remove from cache so they can be initialized properly
 }
 
-export default async (responses) => {
-  const port = await getPort()
-  const app = express()
-  if (responses) {
-    if (responses.structured) app.get(structuredURL, wrap(responses.structured))
-    if (responses.trace) console.log('TRACE NOT IMPLEMENTED!')
-    if (responses.search) console.log('SEARCH NOT IMPLEMENTED!')
-    if (responses.route) app.get(routeURL, wrap(responses.route))
+const bootstrapUtil = async (responses) => {
+  let port
+  let server
```

Comment:
Don't use lets, `server` does not need to be a let, `getPort()` can return the port depending on the env.